### PR TITLE
Fix inventory column  width issues to make their contents readable

### PR DESCRIPTION
**Pull request by AlexSCorey**
##### SUMMARY
This PR addresses issue https://github.com/ansible/awx/issues/3421.

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
 - UI

##### AWX VERSION
```
3.0.1
```

##### ADDITIONAL INFORMATION
Screen Shot 2019-03-15 at 3 45 47 PM

### CODE REVIEW

**Comment by AlexSCorey** [comment]
recheck

File: awx/ui/client/legacy/styles/lists.less [hunk 1]
```diff
@@ -587,6 +587,11 @@ table, tbody {
 }
 
 .List-staticColumnLayout--groups {
+    display: grid;
+    grid-template-columns: 5px 25px auto;
+}
+
+.List-staticColumnLayout--inventoryGroups {
     display: grid;
```
Comment (jlmitch5) [inline awx/ui/client/legacy/styles/lists.less:595]: the `display: grid` line is redundant, you can remove it.

**Review by jlmitch5 — COMMENTED** [review]
looks good, just remove the redundant line

File: awx/ui/client/legacy/styles/lists.less (hunk 1 above)
Comment (AlexSCorey) [reply awx/ui/client/legacy/styles/lists.less:595]: @jlmitch5 good catch.  I have removed it.

**Review by jlmitch5 — APPROVED** [review]
awesome

**Pull request merged**